command/kv_get.go

```diff
@@ -124,7 +124,13 @@ func (c *KVGetCommand) Run(args []string) int {
 	// Parse the paths and grab the KV version
 	if mountFlagSyntax {
 		// In this case, this arg is the secret path (e.g. "foo").
+		origPath := args[0]
 		partialPath = sanitizePath(args[0])
+
+		if hasTrailingSpace(origPath) && !hasTrailingSlash(origPath) {
+			partialPath = partialPath + " "
```

Comment:
Will this work if there are multiple trailing spaces? Maybe a safer option would be to not fully sanitize the path in case of the `kv get`?